flink-connector-kinesis/src/main/java/org/apache/flink/streaming/connectors/kinesis/proxy/KinesisProxy.java

```diff
@@ -498,18 +514,18 @@ private List<StreamShardHandle> getShardsOfStream(
      * subtask's fetcher. This jitter backoff approach will help distribute calls across the
      * fetchers over time.
      *
-     * @param streamName the stream to describe
+     * @param streamArn the stream to describe
      * @param startShardId which shard to start with for this describe operation (earlier shard's
      *     infos will not appear in result)
      * @return the result of the describe stream operation
      */
     private ListShardsResult listShards(
-            String streamName, @Nullable String startShardId, @Nullable String startNextToken)
+            String streamArn, @Nullable String startShardId, @Nullable String startNextToken)
             throws InterruptedException {
         final ListShardsRequest listShardsRequest = new ListShardsRequest();
         if (startNextToken == null) {
             listShardsRequest.setExclusiveStartShardId(startShardId);
-            listShardsRequest.setStreamName(streamName);
+            listShardsRequest.setStreamARN(streamArn);
```

Comment:
Does this impact the DDB streams adapter? I assume it is compatible with this SDK?